From c32dbf3f8772bf15b6fcf4a4b0981933c43b5659 Mon Sep 17 00:00:00 2001
From: "Wang, ArvinX" <arvinx.wang@intel.com>
Date: Fri, 15 Mar 2019 18:13:13 +0800
Subject: [PATCH] Add runtime permission request

<uses-permission android:name="android.permission.GRANT_RUNTIME_PERMISSIONS" />
<uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL" />

Change-Id: Ieebc6296c7c55e2817298bda63c1a509cdbb3ee2
Tracked-On: https://jira.devtools.intel.com/browse/OAM-76346
Signed-off-by: Wang, ArvinX <arvinx.wang@intel.com>
---
 AndroidManifest.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index a8766e4..5046e7e 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -23,6 +23,8 @@
     <!-- Permission to allow the LensPicker to set a default activity for an Intent. -->
     <uses-permission android:name="android.permission.SET_PREFERRED_APPLICATIONS" />
     <uses-permission android:name="android.permission.EXPAND_STATUS_BAR" />
+    <uses-permission android:name="android.permission.GRANT_RUNTIME_PERMISSIONS" />
+    <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL" />
 
     <application
         android:label="@string/app_label" >
-- 
1.9.1

